app/chat-history/chatHistory.tsx

```diff
@@ -0,0 +1,163 @@
+
+
+export default function Chathistory() {
+    const y=Dimensions.get("screen").height;
+    const [selected, setSelected] = useState("Messages");
+    const [isModalVisible, setIsModalVisible] = useState(false);
+    const [visible, setVisible] = useState(false);
+    const [docName, setDocname] = useState('');
+    const handleDocName = (doc: string) => {
+        setDocname(doc);
+        setIsModalVisible(true);
+    }
+    const renderItems = ({ item }: { item: MessagesType }) => {
+        return (
+            <View className="flex flex-row w-1/1 justify-between items-center py-3 ">
+                <View className="flex-none w-[20%]">
+                    <Image source={item.image} style={{ width: 58, height: 58 }} />
+                </View>
+                <TouchableOpacity className="flex-auto w-[73%] pl-2" onPress={() => handleDocName(item.name)}>
+                    <Text className="font-UrbanistBold text-[16px] pb-2">{item.name}</Text>
+                    <Text className="font-UrbanistRegular text-grey">{item.message}</Text>
+                </TouchableOpacity>
+                <View className="flex-auto w-[30%] justify-start">
+                    <Text className="font-UrbanistRegular text-grey pb-2 text-right">{item.date}</Text>
+                    <Text className="font-UrbanistRegular text-grey text-right">{item.time}</Text>
+                </View>
+            </View>
+        )
+    }
+    return (
+        <>
+            <Modal visible={isModalVisible}>
+                <View className={`${visible? '': 'hidden'} absolute w-[100%] right-0 `}>
+                    <View className="w-[100%]">
+                        <View className="w-[35%] relative top-[50px] left-[59%] p-4  pl-2 right-0 bg-white rounded-xl">
+                            <TouchableOpacity className="flex-row gap-2">
+                                <SvgXml xml={deleteBtn} />
+                                <Text className="font-UrbanistRegular">clear chat</Text>
+                            </TouchableOpacity>
+                            <View className="ml-3 pt-1 border-b-[0.5px] border-grey opacity-40"/>
+                            <TouchableOpacity className="flex-row gap-2 py-1">
+                                <SvgXml xml={download} />
+                                <Text className="font-UrbanistRegular" > export chat</Text>
+                            </TouchableOpacity>
+                            <View className="ml-3 pb-1 border-b-[0.5px] border-grey opacity-40"/>
+                            <TouchableOpacity className="flex-row gap-2 pt-2" onPress={()=>{setVisible(false)}} >
+                                <SvgXml xml={deleteRed} stroke={'#F75555'} />
+                                <Text className="font-UrbanistRegular text-[#F75555]">Delete chat</Text>
+                            </TouchableOpacity>
+                            <View className="ml-3 pt-1 border-b-[0.5px] border-grey opacity-40"/>
+                        </View>
```

Comment:
Nice work on this dropdown. Some suggestions would be adding some space between the elements and removing the border on the last element.

Reply:
done and thanks a lot for your comments It was great to work on your feedback, you can test it now if any other thing I can change tell me please🤝